openshift/templates/visual-verifier/visual-verifier-deploy.safe-entry-c19.prod.param

```diff
@@ -20,7 +21,8 @@ TAG_NAME=prod
 # DJANGO_SECRET_KEY=[\w]{50}
 # DJANGO_DEBUG=False
 # VERIFIER_NAME=Essential Service Safe Entry
-OIDC_RP_PROVIDER_ENDPOINT=https://a2a-vc-authn-controller-test.pathfinder.gov.bc.ca
+OIDC_RP_PROVIDER_ENDPOINT=https://a2a-vc-authn-controller-test\.pathfinder.gov.bc.ca
```

Comment:
Invalid URL.  Similar issue in other places.

Reply:
I've removed the URLs now so they'll be populated by the overrides script